login/package.json

```diff
@@ -11,16 +11,19 @@
   "license": "ISC",
   "dependencies": {
     "body-parser": "^1.19.0",
-    "dotenv": "^8.1.0",
+    "dotenv": "^8.2.0",
+    "email-validator": "^2.0.4",
     "express": "^4.17.1",
+    "express-validator": "^6.2.0",
```

Comment:
Are you sure this import is needed, I thought you didn't use express validation.